packages/web/src/routes/apps/preview.tsx

```diff
@@ -56,11 +56,11 @@ function Preview() {
     case 'stopped':
       return (
         <div className="flex justify-center items-center w-full h-full">
-          {lastStoppedError === null ? (
-            <span className="text-tertiary-foreground">Stopped preview server.</span>
+          {exitCode === null || exitCode === 0 ? (
```

Comment:
I improved this a bit here: https://github.com/srcbookdev/srcbook/commit/e94f15791ea9b6d57f9f174f0c3d3cd07a1dfaea